packages/core/documentation/README.md

```diff
@@ -0,0 +1,237 @@
+# @kong-ui-public/documentation
+
+A set of Kong UI components for displaying documents.
+
+- [Features](#features)
+- [Requirements](#requirements)
+- [Components](#components)
+- [Usage](#usage)
+  - [Install](#install)
+
+## Features
+
+- List of package features
+
+## Requirements
+
+- `vue` must be initialized in the host application
+- `@kong/kongponents` must be available as a `dependency` in the host application, along with the package's style imports. [See here for instructions on installing Kongponents](https://kongponents.konghq.com/#globally-install-all-kongponents).
+
+## Components
+
+- `DocumentationContent`
+  - Props
+    - `cacheKey`
+      - Type: Number
+      - Required: false
+      - Default: 0
+      - Use: Used to make `key` attribute dynamic for the `KTreeList` and `DocumentationDisplay` components. The key special attribute is primarily used as a hint for Vue's virtual DOM algorithm to identify vnodes when diffing the new list of nodes against the old list.
+    - `canEdit`
+      - Type: Function
+      - Required: false
+      - Default: async () => false
+      - Use: Permission check for the `DocumentationDisplay` component for allowing CRUD actions
+    - `documentList`
+      - Type: Array
+      - Required: true
+      - Default: N/A
+      - Use: The list of document items passed into the `items` prop of the `KTreeList` component
+    - `documentAst`
+      - Type: Object
+      - Required: true
+      - Default: N/A
+      - Use: Used in the `DocumentationDisplay` component which uses it to create the document object passed to the `DocumentViewer` component
+    - `documentStatus`
+      - Type: String
+      - Required: true
+      - Default: N/A
+      - Use: Used in the `DocumentationDisplay` component to display and update the publish status of the document
```

Comment:
Do these actually need to be separate props?

Reply:
fixed